### PR TITLE
Add Sqlpp11

### PR DESCRIPTION
**Pull request by xsacha**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/xsacha/hunter-h4uji/builds/23143638
  * https://travis-ci.org/xsacha/hunter/builds/507650712

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/344

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/827
---

Notes: Upstream package config was named with a capital 'S', so I went with this. Also, they used no namespace and I kept this like that because I wasn't sure whether the namespace would be a lowercase s or uppercase S. Please advise if you want a namespace.

### CODE REVIEW

File: examples/Sqlpp11/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-sqlpp11)
+
+hunter_add_package(Sqlpp11)
```
Comment (ruslo) [inline examples/Sqlpp11/CMakeLists.txt]: > SphinxWarning: /home/travis/build/ruslo/hunter/docs/packages/pkg/Sqlpp11.rst:18:start-after pattern not found: # DOCUMENTATION_START {
Warning, treated as error:
/home/travis/build/ruslo/hunter/docs/packages/pkg/Sqlpp11.rst:18:start-after pattern not found: # DOCUMENTATION_START {
The command "./jenkins.sh" exited with 1.

`DOCUMENTATION_START {` / `DOCUMENTATION_END }` are missing. Please check template file:
- https://github.com/ruslo/hunter/blob/v0.23.133/examples/foo/CMakeLists.txt#L12-L18

Comment (xsacha) [reply examples/Sqlpp11/CMakeLists.txt]: Oops. Didn't know that's where it gets it from.
Thanks.

Comment (ruslo) [reply examples/Sqlpp11/CMakeLists.txt]: Yes, just copy it:
```
[hunter]> cp docs/packages/pkg/foo.rst docs/packages/pkg/hunter_box_1.rst
[hunter]> sed -i 's,foo,hunter_box_1,g' docs/packages/pkg/hunter_box_1.rst
```
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation

File: docs/packages/pkg/Sqlpp11.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    sqlpp11
```
Comment (ruslo) [inline docs/packages/pkg/Sqlpp11.rst]: > packages/pkg/Sqlpp11.rst:11:Sqlpp:["Slop", "Scalp", "Slap", "Slip", "Slippy", "Sloop", "Slope", "Sloppy", "Scallop", "Clop", "Glop", "Scalper", "Skip", "Sleep", "Slurp", "Skippy", "Scalps", "Sculpt", "Clap", "Clip", "Gulp", "Kelp", "Scalp's"]

Should be `Sqlpp`.

Comment (xsacha) [reply docs/packages/pkg/Sqlpp11.rst]: Is that a bug with the spell checker?
Strips numbers from words to be checked but not from the dictionary?

Comment (ruslo) [reply docs/packages/pkg/Sqlpp11.rst]: > Is that a bug with the spell checker?

I guess so.

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23151401
- https://travis-ci.org/ingenue/hunter/builds/507783552?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23151769
- https://travis-ci.org/ingenue/hunter/builds/507789756?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.134

**Pull request merged**